ducktape/command_line/main.py

```diff
@@ -184,19 +204,14 @@ def main():
 
     # Run the tests
     runner = SerialTestRunner(session_context, tests)
-    test_results = runner.run_all_tests()
+    runner.run_all_tests()  # populates a thing with test results
 
     # Report results
     # TODO command-line hook for type of reporter
-    reporter = SimpleStdoutSummaryReporter(test_results)
-    reporter.report()
-    reporter = SimpleFileSummaryReporter(test_results)
-    reporter.report()
-
-    # Generate HTML reporter
-    reporter = HTMLSummaryReporter(test_results)
-    reporter.report()
 
+    result_store = FileSystemResultStore(args.results_root)
+    generate_reports(session_context.session_id, result_store, results_dir)
     update_latest_symlink(args.results_root, results_dir)
-    if not test_results.get_aggregate_success():
-        sys.exit(1)
+
+    # if not test_results.get_aggregate_success():
```

Comment:
Is this just because this is WIP and commented out for testing? Fine if it is, but just want to make sure we don't accidentally leave this commented out.